cfme/tests/test_replication.py

```diff
@@ -329,7 +341,55 @@ def test_replication_global_region_dashboard():
             1.
             2. Dashboard on the Global displays data from the Remote region
     """
-    pass
+    remote_app, global_app = setup_replication
+    remote_provider = provider_app_crud(InfraProvider, remote_app)
+    remote_provider.setup()
+    assert remote_provider.name in remote_app.managed_provider_names, "Provider is not available."
+
+    new_vm_name = fauxfactory.gen_alphanumeric(start="test_rep_dashboard", length=25).lower()
+    vm = create_vm(provider=remote_provider, vm_name=new_vm_name)
+    request.addfinalizer(vm.cleanup_on_provider)
+    data_items = ('EVM: Recently Discovered Hosts', 'EVM: Recently Discovered VMs',
+                  'Top Storage Consumers')
+
+    remote_app_data = global_app_data = {}
```

Comment:
I think this is dangerous. Do you want both of these dicts to store same values?
Example: 
```
>>> remote_app_data = global_app_data = {} 
>>> remote_app_data['var1'] = 'val1'
>>> remote_app_data
{'var1': 'val1'}
>>> global_app_data
{'var1': 'val1'}

```

As you can see if you modify one dict, the other dict also looks the same. 

If this is undesired, you may use,
```remote_app_data, global_app_data = {},{} ```

Reply:
@kedark3  Thank you for review, will update PR accordingly